.azure-pipelines-templates/common.yml

```diff
@@ -31,6 +31,9 @@ jobs:
     - ${{ if eq(parameters.suffix, 'Coverage') }}:
       - template: coverage.yml
 
+    - ${{ if eq(parameters.suffix, 'Coverage') }}:
```

Comment:
Might be worth renaming this flag as it no longer only guards the coverage job?

Reply:
Do you have a good name in mind?